npm/vite-dev-server/src/index.ts

```diff
@@ -4,14 +4,7 @@ const debug = debugFn('cypress:vite-dev-server:vite')
 
 export { StartDevServerOptions }
 
-type DoneCallback = () => unknown
-
-export interface ResolvedDevServerConfig {
-  port: number
-  close: (done?: DoneCallback) => void
-}
-
-export async function startDevServer (startDevServerArgs: StartDevServerOptions): Promise<ResolvedDevServerConfig> {
+export async function startDevServer (startDevServerArgs: StartDevServerOptions): Promise<Cypress.ResolvedDevServerConfig> {
```

Comment:
All dev server modules should have the same API. Eg, either they all should export a named function `startDevServer` (preferred) or they all should have a default export. This also includes `webpack-dev-server`

Reply:
and they do ;)

https://github.com/cypress-io/cypress/blob/develop/npm/webpack-dev-server/src/index.ts#L17

Reply:
Awesome. The [StartDevServer type](https://github.com/cypress-io/cypress/blob/0dfec805f47c0afbbfbb0dbcbab351e6c13e829d/npm/webpack-dev-server/src/startServer.ts#L7) is not the API outlined in the notion document. Can we also make the `webpack-dev-server` signatures consistent with the others?